packages/core/rsc/types.ts

```diff
@@ -31,6 +31,7 @@ export type AIProviderProps<AIState = any, UIState = any, Actions = any> = {
   children: React.ReactNode;
   initialAIState?: AIState;
   initialUIState?: UIState;
+  $ActionTypes?: Actions;
```

Comment:
without this the generic doesn't get bound to anything and the information is lost

Reply:
This is kinda sad that TS requires us to do this but I don’t see other easy workarounds. To hide it away for users, could you make the key a Symbol instead so no one will actually pass that (and hopefully autocompletion ignores it too)?

Reply:
Symbol doesnt seem to work :/ getting unknown return from useActions again...